zebrad/tests/acceptance.rs

```diff
@@ -3228,3 +3228,257 @@ async fn trusted_chain_sync_handles_forks_correctly() -> Result<()> {
 
     Ok(())
 }
+
+/// Test successful block template submission as a block proposal or submission on a custom Testnet.
```

Comment:
Can we move this test to a different PR ?

Reply:
I'd like to keep it here if it doesn't need significant changes.